develop/dev-guide-overview.md

```diff
@@ -1,7 +1,12 @@
 ---
 title: Developer Guide Overview
 summary: Introduce the overview of the developer guide.
-aliases: ['/tidb/dev/connectors-and-apis/','/appdev/dev/','/tidb/dev/dev-guide-outdated-for-laravel']
+aliases:
+  [
+    '/tidb/dev/connectors-and-apis/',
+    '/appdev/dev/',
+    '/tidb/dev/dev-guide-outdated-for-laravel',
+  ]
```

Comment:
Please keep aliases in one line, because when we check out a new docs branch we'll use regular expression to match and delete all aliases, which are not necessary for a new docs version. Putting them in one line makes deleting easy.
```suggestion
aliases: ['/tidb/dev/connectors-and-apis/','/appdev/dev/','/tidb/dev/dev-guide-outdated-for-laravel']
```